types/map4dsdk/map4dsdk-tests.ts

```diff
@@ -0,0 +1 @@
+import * as map4d from "map4d";
```

Comment:
Please add more tests to avoid regressions, following the suggestions found [here](https://github.com/Microsoft/dtslint#write-tests).